src/styles/blocks/card.scss

```
@@ -0,0 +1,91 @@
+.card {
+  transition: transform 300ms;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  box-sizing: border-box;
+  width: 200px;
+  font-size: 12px;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__imac {
+    margin-top: 32px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__description {
+    display: flex;
+    flex-direction: column;
+    margin: 40px 16px 0;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &:hover .card__name {
+    color: #34568b;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $secondary-text-color;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-text-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: 300ms;
+    margin-bottom: 16px;
+    border-radius: 5px;
+    height: 40px;
+    background-color: $links-color;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    text-decoration: none;
+    color: $links-default-color;
+
+    &:hover {
+      background-color: $links-default-color;
+      border: 1px solid $links-color;
+      box-sizing: border-box;
+      color: $links-color;
```

```suggestion
      background-color: $links-default-color;
      border-color: $links-color;
      color: $links-color;
```
It's not a good idea to change box-sizing on hover. Better set if for all elements (*) so you don't need to change it later.
Also, instead of adding border on hover, consider adding a transparent border for the default state of the element and just change its color on hover.
Btw, add transition for these properties too so they change smoothly